app/views/event1/PaymentNatureView.scala.html

```diff
@@ -0,0 +1,50 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.event1.PaymentNature
+@import controllers.event1.routes
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukRadios: GovukRadios,
+    govukButton: GovukButton
+)
+
+@(form: Form[_], waypoints: Waypoints)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("paymentNature.title"))) {
+
+    @formHelper(action = routes.PaymentNatureController.onSubmit(waypoints), 'autoComplete -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form,  errorLinkOverrides = Map("value" -> "value_0")))
+        }
+
+        @govukRadios(
+            RadiosViewModel(
+                field  = form("value"),
+                legend = LegendViewModel(messages("paymentNature.heading")).asPageHeading().withCssClass("govuk-!-margin-bottom-8"),
+                items  = PaymentNature.options
+            )
+        )
+
+        @govukButton(
+            ButtonViewModel(messages("site.continue"))
```

Comment:
can we add id for continue button ?

` ButtonViewModel(messages("site.continue")).withAttribute(("id", "submit"))`